api/src/generalized.rs

```diff
@@ -56,7 +57,7 @@ pub mod model {
         BlankNode(BlankNode<'a>),
         Literal(Literal<'a>),
         Variable(Variable<'a>),
-        Triple(&'a Triple<'a>),
+        Triple(&'a [GeneralizedTerm<'a>; 3]),
```

Comment:
A specific struct or `Triple { subject: &GeneralizedTerm<'a>, predicate: &GeneralizedTerm<'a>, object: &GeneralizedTerm<'a> }` might be nicer but this is fine imho

Reply:
I would rather keep it that way, as I really on being able to index into triples using `usize`s.